wpiutil/src/main/native/thirdparty/json/include/wpi/json_fwd.h

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
Needs a copyright notice at the top of the file, `#pragma once`, and newline at the end of the file.